examples/full_pipeline.rs

```diff
@@ -247,86 +247,92 @@ fn main() {
     //println!("The s value is {}", s.s);
     let args = erdos::new_app("ERDOS").get_matches();
     let mut node = Node::new(Configuration::from_args(&args));
+    let graph = Graph::new();
 
     let source_config = OperatorConfig::new().name("SourceOperator");
-    let source_stream = erdos::connect_source(SourceOperator::new, source_config);
+    let source_stream = graph.connect_source(SourceOperator::new, source_config);
 
     let square_config = OperatorConfig::new().name("SquareOperator");
     let square_stream =
-        erdos::connect_one_in_one_out(SquareOperator::new, || {}, square_config, &source_stream);
-
-    let map_config = OperatorConfig::new().name("FlatMapOperator");
-    let map_stream = erdos::connect_one_in_one_out(
-        || -> FlatMapOperator<usize, _> {
-            FlatMapOperator::new(|x: &usize| std::iter::once(2 * x))
-        },
-        || {},
-        map_config,
-        &square_stream,
-    );
+        graph.connect_one_in_one_out(SquareOperator::new, || {}, square_config, &source_stream);
+
+    // let map_config = OperatorConfig::new().name("FlatMapOperator");
+    // let map_stream = graph.connect_one_in_one_out(
+    //     || -> FlatMapOperator<usize, _> {
+    //         FlatMapOperator::new(|x: &usize| std::iter::once(2 * x))
+    //     },
+    //     || {},
+    //     map_config,
+    //     &square_stream,
+    // );
```

Comment:
Why are these commented out?

Reply:
Not sure, I'll uncomment them as they are working